go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/xanzy/go-gitlab v0.81.0
 	go.etcd.io/bbolt v1.3.7
```

Comment:
@jukie if you get a chance, could you end to end test these changes in your own setup to verify things work as expected?

Reply:
Anything specific? Not sure I understand.
Both things I called out were verified in my environment.
1. Only single parent dir modules would work previously.
2. If using per-repo config, nothing would work.

Reply:
Were you also able to test the case if autoplanning of modules is not enabled?

Reply:
Nope, will check that tonight.

Reply:
Verified by disabling autoplan specifically on a project and globally and things work as intended. Gitlab users still hit the goroutine panic below when disabling autoplan globally but an API plan or other workarounds can be used to verify. That's a separate issue which I believe is already open and I've been trying to figure out on the side.

```
runtime error: invalid memory address or nil pointer dereference
runtime/panic.go:260 (0x44e07c)
runtime/signal_unix.go:837 (0x44e04c)
github.com/runatlantis/atlantis/server/events/vcs/gitlab_client.go:225 (0xd306c8)
github.com/runatlantis/atlantis/server/events/vcs/proxy.go:76 (0xd35fc4)
github.com/runatlantis/atlantis/server/events/vcs/pull_status_fetcher.go:32 (0xd36895)
github.com/runatlantis/atlantis/server/events/plan_command_runner.go:174 (0xf60ab7)
github.com/runatlantis/atlantis/server/events/plan_command_runner.go:290 (0xf6216b)
github.com/runatlantis/atlantis/server/events/command_runner.go:296 (0xf47c83)
runtime/asm_amd64.s:1598 (0x46b220)
```

Reply:
I think that is related to this : https://github.com/runatlantis/atlantis/pull/1981